src/common/transformations/include/ov_ops/convolution_biased.hpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/coordinate_diff.hpp"
+#include "openvino/core/node.hpp"
+#include "openvino/op/op.hpp"
+#include "openvino/op/util/convolution_base.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace op {
+namespace internal {
+
```

Comment:
please add description for this interal op and its purpose.

Reply:
Done